Makefile

```diff
@@ -101,6 +101,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	kubectl apply -f namespace.yaml
```

Comment:
This should be one line above. Namespace creation should happen before the rest of the resources.

Reply:
That makes sense. Fixing it now.
Thanks!